src/AgentContainer.H

```diff
@@ -74,20 +74,27 @@ public:
         : amrex::ParticleContainer< 0,
                                     0,
                                     RealIdx::nattribs,
-                                    IntIdx::nattribs> (a_geom, a_dmap, a_ba)
+                                    IntIdx::nattribs> (a_geom, a_dmap, a_ba),
+          student_counts(a_ba, a_dmap, 6, 0),
+          teacher_counts(a_ba, a_dmap, 6, 0)
     {
         BL_PROFILE("AgentContainer::AgentContainer");
         AMREX_ASSERT(m_num_diseases < ExaEpi::max_num_diseases);
         m_num_diseases = a_num_diseases;
         m_disease_names = a_disease_names;
 
+        student_counts.setVal(0);  // Initialize the MultiFab to zero
+        teacher_counts.setVal(0);
+
         add_attributes();
 
         {
             amrex::ParmParse pp("agent");
             pp.query("symptomatic_withdraw", m_symptomatic_withdraw);
             pp.query("shelter_compliance", m_shelter_compliance);
             pp.query("symptomatic_withdraw_compliance", m_symptomatic_withdraw_compliance);
+            pp.queryarr("teacher_student_ratios", teacher_student_ratios);
```

Comment:
Given how you are using this variable, change it to `student_teacher_ratios` since it represents student to teacher ratio, not the teacher to student ratio.